integration-tests/cli/product-export.it.js

```diff
@@ -0,0 +1,120 @@
+import fs from 'fs'
+import tmp from 'tmp'
+import { getCredentials } from '@commercetools/get-credentials'
+import { exec } from 'child_process'
+import { version } from '@commercetools/product-exporter/package.json'
+import {
+  sampleProductType,
+  sampleTaxCategory,
+  createProducts,
+  expectedProducts,
+} from './helpers/products-export.data'
+import { createData, clearData } from './helpers/utils'
+
+let projectKey
+if (process.env.CI === 'true')
+  projectKey = 'product-export-integration-test'
+else
+  projectKey = process.env.npm_config_projectkey
+
+describe('Product Exporter', () => {
+  let apiConfig
+  const productType = { typeId: 'product-type' }
+  const taxCategory = { typeId: 'tax-category' }
+  const bin = './integration-tests/node_modules/.bin/product-exporter'
+
+  beforeAll(() => getCredentials(projectKey)
+    .then((credentials) => {
+      apiConfig = {
+        host: 'https://auth.sphere.io',
+        apiUrl: 'https://api.sphere.io',
+        projectKey,
+        credentials,
+      }
+      return clearData(apiConfig, 'products')
+    })
+    .then(() => (
+      Promise.all([
+        clearData(apiConfig, 'productTypes'),
+        clearData(apiConfig, 'taxCategories'),
+      ])
+    ))
+    .then(() => (
+      Promise.all([
+        createData(apiConfig, 'productTypes', [sampleProductType]),
+        createData(apiConfig, 'taxCategories', [sampleTaxCategory]),
+      ])
+    ))
+    .then(([
+      createdProductType,
+      createdTaxCategory,
+      ]) => {
+      productType.id = createdProductType[0].body.id
+      taxCategory.id = createdTaxCategory[0].body.id
+
+      const sampleProducts = createProducts(
+        productType,
+        taxCategory,
+      )
+      return createData(apiConfig, 'products', sampleProducts)
+    })
+    .catch(process.stderr.write)
```

Comment:
why are you _catching_ here, I think if an error occurs it's not going to exit out of the test